openfecli/parameters/mol.py

```diff
@@ -0,0 +1,36 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+from plugcli.params import MultiStrategyGetter, Option, NOT_PARSED
+
+
+def _load_molecule_from_smiles(user_input, context):
+    from openfe.setup import Molecule
+    from rdkit import Chem
+    # MolFromSmiles returns None if the string is not a molecule
+    # TODO: find some way to redirect the error messages? Messages stayed
+    # after either redirect_stdout or redirect_stderr.
+    mol = Chem.MolFromSmiles(user_input)
+    if mol is None:
+        return NOT_PARSED
+
+    # TODO: next is (temporary?) hack: see
+    # https://github.com/OpenFreeEnergy/Lomap/issues/4
+    Chem.rdDepictor.Compute2DCoords(mol)
```

Comment:
This fix should probably get pushed back onto the LomapAtomMapper until the bug is fixed.  (Check if any conformers, if not add 2d)

Reply:
Probably a more elegant way but I was doing this at one point:

```
    # Not sure if there is a better way to check if 2DCoords exist
    for mol in [mol_1, mol_2]:
        try:
            mol.GetConformers()[0]
        except IndexError:
            Chem.rdDepictor.Compute2DCoords(mol)
```